.gitignore

```diff
@@ -77,4 +77,6 @@ pom.xml.versionsBackup
 
 ams/ams-server/src/test/java/com/netease/arctic/ams/server/sql/mydb1/*
 conf/
-/site/generated/
\ No newline at end of file
+/site/generated/
+
+*/.gitignore
```

Comment:
I modify the `.gitignore` to support locally customized `.gitignore` files in a subdirectory.